--- a/mopidy_mpd/protocol/playback.py
+++ b/mopidy_mpd/protocol/playback.py
@@ -221,7 +221,7 @@
     tl_tracks = context.core.tracklist.filter({"tlid": [tlid]}).get()
     if not tl_tracks:
         raise exceptions.MpdNoExistError("No such song")
-    return context.core.playback.play(tl_tracks[0]).get()
+    return context.core.playback.play(tl_tracks[0].tlid).get()
 
 
 @protocol.commands.add("previous")
